WalletWasabi.Fluent/Views/StatusBar/StatusBar.axaml

```diff
@@ -87,8 +88,8 @@
             <Button Classes="plain" Command="{Binding UpdateCommand}">
               <StackPanel Spacing="5" Orientation="Horizontal">
                 <PathIcon Data="{StaticResource arrow_download_regular}" Height="13"
-                          Foreground="{StaticResource SystemAccentColor}" />
-                <TextBlock Text="Update now" Foreground="{StaticResource SystemAccentColor}" />
+                          Foreground="{StaticResource StatusForegroundColor}" />
+                <TextBlock Text="Update now" Foreground="{StaticResource StatusForegroundColor}" />
```

Comment:
It doesn't really notice that they are clickable buttons. They seem more like simple status texts.

![image](https://user-images.githubusercontent.com/16364053/160817750-a2408dc4-aed4-4f86-9536-082307b68ca9.png)

Also `Foreground="{StaticResource StatusForegroundColor}"` Doesn't needed... It causes an issue in light mode.

![image](https://user-images.githubusercontent.com/16364053/160819110-4e0cfc09-f955-4cdd-bc28-b98857bbe90d.png)